html/index.html

```diff
@@ -0,0 +1,165 @@
+        <p class="about">
+            A one in a million gathering of leaders from all over the world to ahare their journey and wealth of
+            experience with the next generation is taking place in December, in Kenya.
+        </p>
+        <p class="date">2022.11.17(THU) ~ 18(FRI)</p>
+        <p class="location">
+            @Kenya International Conference Center (KICC), Carnivore and more
+        </p>
+    </section>
+    <section id="about">
+        <section class="main-program aboutPage" id="program">
+            <h3 class="mp-title">Main Program</h3>
+            <hr />
+            <ul>
+                <li class="programs">
+                    <img src="/images/lectureIcon.png" width="35" height="30" alt="" class="img-program" />
+                    <h4>Lecture</h4>
+                    <p class="program-about">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis,
+                        libero cumque voluptatibus repellat
+                    </p>
+                </li>
+                <li class="programs">
+                    <img src="/images/exhibitionIcon.png" width="30" height="30" alt="" class="img-program" />
+                    <h4>Exhibition</h4>
+                    <p class="program-about">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis,
+                        libero cumque voluptatibus repellat
+                    </p>
+                </li>
+                <li class="programs">
+                    <img src="/images/forumIcon.png" width="30" height="30" alt="" class="img-program" />
+                    <h4>Forum</h4>
+                    <p class="program-about">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis,
+                        libero cumque voluptatibus repellat
+                    </p>
+                </li>
+                <li class="programs">
+                    <img src="/images/workshopIcon.png" width="30" height="30" alt="" class="img-program" />
+                    <h4>Workshop</h4>
+                    <p class="program-about">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis,
+                        libero cumque voluptatibus repellat
+                    </p>
+                </li>
+                <li class="programs">
+                    <img src="/images/networkingIcon.png" width="30" height="30" alt="" class="img-program" />
+                    <h4>Network</h4>
+                    <p class="program-about">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis,
+                        libero cumque voluptatibus repellat
+                    </p>
+                </li>
+            </ul>
+            <a class="join none"> Join LT Global Summit 2022 </a>
+            <p class="see-projects none1"><a>SEE THE WHOLE PROJECTS</a></p>
+        </section>
```

Comment:
Please try to match the main program section as on the [figmal design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
### **Figmal Design**
![figMain](https://user-images.githubusercontent.com/55596970/181652819-f14c6d2b-bc42-4d6d-bca9-f3b72c2c2a44.png)
### **Your Design**
![YourMain](https://user-images.githubusercontent.com/55596970/181652928-2eac778b-2c5d-4600-bdb0-bea2e976665d.png)